### PR TITLE
VMRCM Disk Debug

### PR DESCRIPTION
**Pull request by meisonlikesicecream**
TL;DR: fixed some bugs for the VMRCM in disk as that I hadn't noticed before.

Up until now, I hadn't been able to properly check the output of the VMRCM in the disks as the testvectors contained the stubs in a different order from what the HLS VMRCM was processing them in. So even though the VMRCM disk was running fine, it was not possible to compare them to the testvectors. Managed to get some new testvectors from Anders where they were in the correct order, and could therefore finally check the output from VMRCM in the disks. Noticed a few bugs related to some bit-widths that I have fixed in this PR.

I have not updated the testvectors in this PR, I guess that'll happen sometime in the future, so if someone wants to run it themselves (without failing the simulation as it currently does, and will continue to do until we have updated the testvectors) they'll have to use the following in `download.sh`:
```
# Combined modules
memprints_url_cm="https://aryd.web.cern.ch/aryd/MemPrints_Combined_211012.tgz"
luts_url_cm="https://aryd.web.cern.ch/aryd/LUTs_Combined_211012.tgz"
```

### CODE REVIEW

**Review by aehart — APPROVED** [review]
Looks good. Approved!
